test/val/val_decoration_test.cpp

```diff
@@ -360,5 +360,74 @@ TEST_F(ValidateDecorations, EntryPointFunctionHasLinkageAttributeBad) {
                 "an OpEntryPoint instruction."));
 }
 
+TEST_F(ValidateDecorations, FunctionDeclarationWithoutImportLinkageBad) {
```

Comment:
I'd like to see a test where we exercise the fact you got the "1" value right for the linkage attribute.
It's the same as this case, except you add an *Export* linkage decoration to the function.